README.md

```diff
@@ -34,14 +34,60 @@ Integrate with your application using:
 const MarketplaceTx = require('marketplace-tx');
 ```
 
+#### Using in browser
+The library uses ES2017 features therefore it needs to be transpiled with Babel before it can be used in browser.
```

Comment:
Perhaps replace with 'may need'? In fact most browser support async-await, so it depends on your babel settings - if you are using babel, you need to use the env preset (or similar).